### DISCUSSION TITLE
Can't Change HA Attributes

### DISCUSSION THREAD

**Discussion by martoq** [zachowj/hass-node-red#84] [Q&A] [answered]
Having an issue with the HA Portion of a new node configuration.  I searched the discussion and was unable to find an answer.  No matter what I set the node type to the HA Configuration (optional) portion is greyed out.  The closest I was able to do was go into integrations and change the entity id there to something user friendly but from my understanding that's what should be done on the node-red side or am I misunderstanding the purpose of the Optional config.

![image](https://user-images.githubusercontent.com/11802571/144447739-71828e7a-ce55-4d8d-838d-c096b7528674.png)
![image](https://user-images.githubusercontent.com/11802571/144447873-9b77463c-1baa-44d9-b659-5f2bd76a828d.png)

**Comment by zachowj** [comment, marked as answer]
The property name is not "disabled" it's just not editable and should still function correctly. The entity id is only set the first deployment after that it will only change the friendly name.

<https://zachowj.github.io/node-red-contrib-home-assistant-websocket/guide/custom_integration/#tip>

**Reply by martoq** [reply]
Appreciate the clarification thank you!
